app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,55 @@
+class TasksController < ApplicationController
+
+  def index
+    @tasks = Task.all.order(:status)
+  end
+
+  def new
+    @task = Task.new
+  end
+
+  def create
+    task = Task.new(task: params[:task][:task], description: params[:task][:description], due: params[:task][:due], status: false)
+    task.save
+    redirect_to('/tasks')
+  end
+
+  def show
+    @task = Task.find(params[:id])
+  end
+
+  def edit
+    @task = Task.find(params[:id])
+  end
+
+  def all
```

Comment:
This action seems to be doing the same thing as the `index` action. Is there a reason to have this one as well?